lib/trento/settings.ex

```diff
@@ -154,6 +155,38 @@ defmodule Trento.Settings do
     end
   end
 
+  # Analytics settings
+
+  @spec get_analytics_settings() ::
+          {:ok, AnalyticsSettings.t()} | {:error, :analytics_settings_not_configured}
```

Comment:
Change the return options to `{:error, :settings_not_configured}`.
If for some reason you need a specific value, you will need to handle this in the fallback controller as well.
I think the generic one is good enough

Reply:
This has been added to the fallback controller now because `:settings_not_configured` only refers to SUMA.